distributed/protocol/pickle.py

```diff
@@ -5,7 +5,12 @@
 
 if sys.version_info.major == 2:
     import cPickle as pickle
-else:
+elif sys.version_info.major == 3 and sys.version_info.minor < 8:
+    try:
+        import pickle5 as pickle
+    except ImportError:
+        import pickle
+elif sys.version_info.minor:
```

Comment:
Why this condition?

Reply:
I think this was bad copy-pasta. Probably should be `else`.